solr/core/src/java/org/apache/solr/util/PackageTool.java

```diff
@@ -149,7 +149,8 @@ protected void runImpl(CommandLine cli) throws Exception {
                 String version = parsedVersion.second();
                 boolean noprompt = cli.hasOption('y');
                 boolean isUpdate = cli.hasOption("update") || cli.hasOption('u');
-                packageManager.deploy(packageName, version, PackageUtils.validateCollections(cli.getOptionValue("collections").split(",")), cli.getOptionValues("param"), isUpdate, noprompt);
+                String collections[] = cli.hasOption("collections")? PackageUtils.validateCollections(cli.getOptionValue("collections").split(",")): new String[] {};
```

Comment:
Please don't use C-style array declarations.  IMO our pre-commit ought to be enhanced to not allow this